airflow/jobs/base_job.py

```diff
@@ -171,17 +171,14 @@ def heartbeat(self):
             if self.state == State.SHUTDOWN:
                 self.kill()
 
-            is_unit_test = conf.getboolean('core', 'unit_test_mode')
```

Comment:
Does anything else actually use this setting anymore, or could we remove it now?

Reply:
This is still used in other tests unfortunately. But this PR should setup the example on how to remove or avoid using it in other places going forward.